SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -221,6 +221,17 @@ void USpatialGDKEditorSettings::SetPrimaryRegionCode(const ERegionCode::Type Reg
 	PrimaryDeploymentRegionCode = RegionCode;
 }
 
+void USpatialGDKEditorSettings::SetAssemblyWindowsPlatform(const FString& Platform)
+{
+	AssemblyWindowsPlatform = Platform;
+	SaveConfig();
```

Comment:
do we need to do a SaveConfig() here?

Reply:
Unclear to me what the rules for SaveConfig are, I see it is used but not always. Having SaveConfig here ensures that the values are persisted when they are changed. This is the safest behavior.

Reply:
Because we're changing a config value from code, it won't be saved (and thus won't be loaded the next time you open the editor) unless we do this.